lib/diff/data.js

```diff
@@ -18,13 +19,20 @@ var plugin = function(diff) {
 
     diff.dataProvider('$mongodb', function(callback) {
         var mongodb
-        // TODO: add code to connect to a mongodb server
-        
-        // TODO: return the server object 
-        // callback(null, mongodb)
-        
-        callback('$mongodb has not been implemeneted')
-    })
+        var mongoClient = require('mongodb').MongoClient;
+	mongoClient.connect("mongodb://localhost:27017/fcq", function (err, db) {
+	    if(!err){
+		debug("Connected to fcq database")
```

Comment:
There are a few spots where theindentation/styling is weird. IIRC you use VIM... there's a quick way to auto-indent a section:

1. Select the section you want to indent in visual mode (Ctrl-V)
2. Press `=`
3. Voila!